release/README.md

```diff
@@ -18,19 +18,24 @@
   git checkout main && git pull
   ```
 * Update the CHANGELOG.md with the corresponding release notes and open a PR
-  * Use a comparison such as https://github.com/earthly/earthly/compare/v0.3.0...main (replace the versions in the URL with the previously released version) to see which PRs will go into this release.
+  * Use a comparison such as https://github.com/earthly/earthly/compare/v0.7.19...main (replace the versions in the URL with the previously released version) or a tool such as `gitk` (aka `git-gui`) to see which PRs will go into this release.
 * Make sure that main build is green for all platforms (check build status for the latest commit on GitHub).
 * Make sure nightly BuildKite builds are green for most recent build
   | Platform      | Status        |
   | ------------- | ------------- |
   | MacOS (x86)   | [![Build status](https://badge.buildkite.com/cc0627732806ab3b76cf13b02c498658b851056242ec28f62d.svg)](https://buildkite.com/earthly-technologies/earthly-mac-scheduled)
   | MacOS (M1)    | [![Build status](https://badge.buildkite.com/10a7331b2032fcc9f7f311c5218d12c1a18c317cd7fc9270ba.svg)](https://buildkite.com/earthly-technologies/earthly-m1-scheduled)
   | Windows (WSL) | [![Build status](https://badge.buildkite.com/19d9cf7fcfb3e0ee45adcb29abb5ef3cfcd994fba2d6dc148c.svg)](https://buildkite.com/earthly-technologies/earthly-windows-scheduled)
-* Run
-  ```bash
-  env -i HOME="$HOME" PATH="$PATH" SSH_AUTH_SOCK="$SSH_AUTH_SOCK" RELEASE_TAG="$RELEASE_TAG" USER="$USER" PRERELEASE="$PRERELEASE" ./release.sh
-  ```
-* Merge branch `main` into `docs-0.7`
+  * Run
+    ```bash
+    cd release
```

Comment:
I don't think this is needed -- the README is already in the `release` directory.

Reply:
The readme is, but for example when I was using the readme I opened it in my browser and my terminal was pointing to the repo root. also other commands such as `./earthly` assume your terminal is in root.

Reply:
ah that's a good point regarding `./earthly`, feel free to ignore this comment then.